includes/tutorials-editors-data-binding.md

```diff
@@ -0,0 +1,6 @@
+The {WidgetName} can load data from different data source types. To use a local array, assign it to the [dataSource](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#dataSource) property. If array elements are objects, set the fields that supply the {WidgetName}'s value ([valueExpr](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#valueExpr)) and displayed value ([displayExpr](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#displayExpr)). For information on other data source types, refer to the following articles:
```

Comment:
>  If array elements are objects, **set the fields** that supply the {WidgetName}'s value ([valueExpr](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#valueExpr)) and displayed value ([displayExpr](/Documentation/ApiReference/UI_Components/dx{WidgetName}/Configuration/#displayExpr)).

How should I set them? Or is it that obvious?

Reply:
This text is exported to two tutorials - GS with SelectBox and with TagBox where we show code after this text. Also this is quite obvious for this component type.

Reply:
Alright. If this is as obvious as you say, I have no further questions or objections.